internal/Position.go

```diff
@@ -0,0 +1,7 @@
+package internal
+
+type Position struct {
+	ID     string  `json:"id"`
+	Name   string  `json:"name"`
+	Salary float64 `json:"salary"`
```

Comment:
Check fields from the Go model with model declared in OpenAPI specification

Reply:
You tell me about "ID" field?

Reply:
`ID` filed and type of `Salary` filed

Reply:
I have string "ID" field in documentation, what I should change in "ID"?